### PR TITLE
131 literature table block

### LINKED ISSUE #131: Research literature page - Table block fixes
literature page needs to use a table block to render several 2-column tables with headings. This block needs some fixes to match the experience.

Live site: https://www.sunstar.com/research/literature
Franklin: https://main--sunstar--hlxsites.hlx.page/_drafts/mohit/research/literature
image

### PR DESCRIPTION
**Pull request by mkbansal1**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fixes #131 

Test URLs:
- Original: https://www.sunstar.com/research/literature
- Before: https://main--sunstar--hlxsites.hlx.live/_drafts/mohit/research/literature
- After: https://131-literature-table-block--sunstar--hlxsites.hlx.page/_drafts/mohit/research/literature

- [x]  **If you are adding a new block or a variation to an existing block**, please fill below:

  Block library path: https://main--sunstar--hlxsites.hlx.page/tools/sidekick/library.html?plugin=blocks&path=/sidekick/blocks/tables&index=4

### CODE REVIEW

**Review by sdmcraft — COMMENTED** [review]
There seems to be some horizontal scroll on the page.

https://github.com/hlxsites/sunstar/assets/1191451/8cf732ae-2308-4d42-9ca8-18c1ce311d2b

**Comment by mkbansal1** [comment]
> There seems to be some horizontal scroll on the page.
> 
>  Screen.Recording.2023-10-16.at.1.09.04.PM.mov

@sdmcraft This scroll is introduced by Tabs rendered by hero-horizontal-tabs block. I have raised a card to track this issue
https://github.com/hlxsites/sunstar/issues/151

File: blocks/table/table.css [hunk 1]
```diff
@@ -144,4 +132,96 @@ main .multiple-table .flat-table {
...
+
+main .table.text-table table tr td a .icon {
+    margin-left: 0.25em;
+    height: 20px;
+    width: 24px;
+}
+
+main .table.text-table table tr td:first-child {
+    color: var(--transparent-grey-color);
+    font-size: 16px;
```
Comment (sdmcraft) [inline blocks/table/table.css:203]: Please global style pattern. There must be a variable for this in scripts.js

File: blocks/table/table.css [hunk 2]
```diff
@@ -144,4 +132,96 @@ main .multiple-table .flat-table {
...
+}
+
+main .table.text-table table tr td a {
+    font-size: 1rem;
+    line-height: 2rem;
+}
+
+main .table.text-table table tr td a .icon {
+    margin-left: 0.25em;
+    height: 20px;
```
Comment (sdmcraft) [inline blocks/table/table.css:197]: Nit: Prefer `rem` usints based styling here and elsewhere in this PR.

File: blocks/table/table.css [hunk 3]
```diff
@@ -144,4 +132,96 @@ main .multiple-table .flat-table {
...
+    font-weight: 700;
+    line-height: 34px;
+}
+
+main .table.text-table table tr td p {
+    font-size: 14px;
+}
+
+main .table.text-table table tr td a {
+    font-size: 1rem;
```
Comment (sdmcraft) [inline blocks/table/table.css:191]: Please global style pattern. There must be a variable for this in scripts.js

File: blocks/table/table.css [hunk 4]
```diff
@@ -144,4 +132,96 @@ main .multiple-table .flat-table {
...
+
+main .table.text-table table tr td {
+    color: var(--text-color);
+    flex-basis: calc(100% - 7.5em);
+}
+
+main .table.text-table table tr td h4 {
+    color: var(--text-color);
+    font-size: 20px;
+    font-weight: 700;
```
Comment (sdmcraft) [inline blocks/table/table.css:182]: Please global style pattern. There must be a variable for this in scripts.js

**Review by sdmcraft — COMMENTED** [review]
LGTM. Added few comments. We should be good to merge after addressing them.

**Comment by mkbansal1** [comment]
@sdmcraft This PR got merge conflicts, i have raised a separate PR with the recommended changes

https://github.com/hlxsites/sunstar/pull/153

**Pull request closed without merging**